estimator/Pipfile

```diff
@@ -12,3 +12,8 @@ python_version = "3.7"
 black = "==19.10b0"
 pylint = "*"
 twine = "*"
+pytest = "*"
+ipython = "*"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Is there a need for prereleases here? Unless we really rely on some cutting edge features, I don't think we'd generally want them.

Reply:
This was added automatically when I enabled Black.

Reply:
Ah, interesting. It's not needed though, since we fix the version of `black`. (Did I mention they should make a proper release already?)